commons/src/main/kotlin/com/simplemobiletools/commons/helpers/BaseConfig.kt

```diff
@@ -195,6 +197,26 @@ open class BaseConfig(val context: Context) {
 
     fun getFolderProtectionType(path: String) = prefs.getInt("$PROTECTED_FOLDER_TYPE$path", PROTECTION_NONE)
 
+    var lastCopyPath: String
+        get() = prefs.getString(LAST_COPY_PATH, "")!!
+        set(lastCopyPath) = prefs.edit().putString(LAST_COPY_PATH, lastCopyPath).apply()
+
+    fun getDefaultCopyDestinationPath(showHidden: Boolean, currentPath: String): String {
```

Comment:
move this to some context extension please, it doesnt fit here at all

Reply:
moved to [Context-storage.kt](https://github.com/SimpleMobileTools/Simple-Commons/pull/1713/files#diff-108531421030a753f4c62b949716f1e4e1c2262c072d8f2ece16730a1e4b8aadR995)